app/components/common/userInput/index.tsx

```diff
@@ -24,7 +24,7 @@ const UserInput: FC<UserInputProps> = ({
   setValue,
   description,
   err,
-  name
+  name,
```

Comment:
why?

Reply:
seems like this is done by the formatter

Reply:
Yes, it has been done by formatter